hip_data_tools/etl/adwords_to_s3.py

```diff
@@ -196,6 +199,15 @@ def _get_adwords_util(self) -> AdWordsReportReader:
                 conn=GoogleAdWordsConnectionManager(self.__settings.source_connection_settings))
         return self._adwords_util
 
+    def _mask_field_types(self, df: DataFrame):
+        field_type_transformer = FieldTypeTransformer(df=df)
+        for field_name in df.columns:
+            field_type = self.__settings.transformation_field_type_mask[field_name]
+            if field_type is datetime:
+                df[field_name] = df[field_name].astype('datetime64[ns]')
+            else:
+                field_type_transformer.transform(field_name, field_type)
+
```

Comment:
Iterate over the mask fields:
  - if type is int
     - requires special handling do that
  - apply df.as_type

Reply:
done